From 9300a554a1a88211f4bd1415b5b9c2edb4f57d21 Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
Date: Fri, 19 Oct 2012 21:35:45 +0300
Subject: [PATCH 049/518] dpaa_eth: Remove unused ingress_calls field.

There is already the percpu_priv_s->stats->rx_packets field, no need for
the extra ingress_calls.

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 drivers/net/ethernet/freescale/dpa/dpaa_eth.c |    8 +++-----
 drivers/net/ethernet/freescale/dpa/dpaa_eth.h |    1 -
 2 files changed, 3 insertions(+), 6 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth.c b/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
index 05c8aa5..b8af061 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth.c
@@ -2131,8 +2131,6 @@ ingress_rx_default_dqrr(struct qman_portal		*portal,
 		return qman_cb_dqrr_stop;
 	}
 
-	prefetchw(&percpu_priv->ingress_calls);
-
 	/* Vale of plenty: make sure we didn't run out of buffers */
 	dpaa_eth_refill_bpools(percpu_priv);
 	_dpa_rx(net_dev, priv, percpu_priv, &dq->fd, fq->fqid);
@@ -2995,7 +2993,7 @@ static int __cold dpa_debugfs_show(struct seq_file *file, void *offset)
 			dpa_bp_count = *percpu_priv->dpa_bp_count;
 
 		total.in_interrupt += percpu_priv->in_interrupt;
-		total.ingress_calls += percpu_priv->stats.rx_packets;
+		total.stats.rx_packets += percpu_priv->stats.rx_packets;
 		total.stats.tx_packets += percpu_priv->stats.tx_packets;
 		total.tx_returned += percpu_priv->tx_returned;
 		total.tx_confirm += percpu_priv->tx_confirm;
@@ -3019,10 +3017,10 @@ static int __cold dpa_debugfs_show(struct seq_file *file, void *offset)
 				percpu_priv->l4_hxs_errors,
 				dpa_bp_count);
 	}
-	seq_printf(file, "Total     %8u  %8u  %8lu  %8u  %8u  %8u  %8lu  %8lu" \
+	seq_printf(file, "Total     %8u  %8lu  %8lu  %8u  %8u  %8u  %8lu  %8lu"\
 				"     %8u    %8d\n",
 			total.in_interrupt,
-			total.ingress_calls,
+			total.stats.rx_packets,
 			total.stats.tx_packets,
 			total.tx_returned,
 			total.tx_confirm,
diff --git a/drivers/net/ethernet/freescale/dpa/dpaa_eth.h b/drivers/net/ethernet/freescale/dpa/dpaa_eth.h
index 1680b86..da91e24 100644
--- a/drivers/net/ethernet/freescale/dpa/dpaa_eth.h
+++ b/drivers/net/ethernet/freescale/dpa/dpaa_eth.h
@@ -335,7 +335,6 @@ struct dpa_percpu_priv_s {
 	int skb_count;
 #endif
 	u32 in_interrupt;
-	u32 ingress_calls;
 	u32 tx_returned;
 	u32 tx_confirm;
 	/* fragmented (non-linear) skbuffs received from the stack */
-- 
1.7.5.4

